src/main/java/io/ebeaninternal/server/core/PersistRequestBean.java

```diff
@@ -736,7 +736,7 @@ public void setBoundId(Object idValue) {
   @Override
   public final void checkRowCount(int rowCount) {
     if (ConcurrencyMode.VERSION == concurrencyMode && rowCount != 1) {
-      String m = Message.msg("persist.conc2", "" + rowCount);
+      String m = Message.msg("persist.conc2", String.valueOf(rowCount));
```

Comment:
("" + primitive) is not a performant way to get a string representation of said primitive.